src/main/scala/org/apache/mesos/chronos/scheduler/mesos/ConstraintChecker.scala

```diff
@@ -0,0 +1,33 @@
+package org.apache.mesos.chronos.scheduler.mesos
+
+import org.apache.mesos.Protos
+import org.apache.mesos.chronos.scheduler.jobs.constraints.Constraint
+import java.util.logging.Logger
+import scala.collection.JavaConverters._
+
+/**
+ * Helper for checking resource offer against job constraints
+ */
+class ConstraintChecker {
```

Comment:
Instance of this class hold no state, so I think this should be turned into a singleton object (`object ConstraintChecker {`)